### DISCUSSION TITLE
Embedded links to Github not working anymore

### DISCUSSION THREAD

**Discussion by dgafka** [GitbookIO/community#264] [0. General bugs and questions]
### Ask a question, start a conversation, or make an annoucement

Previously links to Github repositories could be copy pasted and were shown inside the box. 
However currently all links which were posted like this are no longer rendering correctly, which means all the links are unavailable for end users.

![Screenshot from 2023-07-10 07-59-45](https://github.com/GitbookIO/community/assets/6060791/6e542b73-e6d8-4a45-b6ae-7d37dc2adf0a)

Example in existing documentation: https://docs.ecotone.tech/quick-start-php-ddd-cqrs-event-sourcing/php-cqrs

**Comment by addisonschultz** [comment]
Super sorry to hear this happened - This is happening with all of the links being used in this way? Can you let me know if re-pasting them works?

**Reply by dgafka** [reply]
Hey, 
Yep it does for all the links, new links are failing too

**Reply by addisonschultz** [reply]
Can you share an example of ones not working? Are they private at all? 

I'd like to test it on my end - not able to reproduce this one so far

**Reply by dgafka** [reply]
This link for example: "https://github.com/ecotoneframework/quickstart-examples/tree/main/CQRS".

If I copy/pasted that into different space, it worked well, however on the linked space it does not work for either old or new links.

**Reply by addisonschultz** [reply]
@dgafka Been debugging this on my end this morning - can you re-authorize the app, and let me know if it's working afterwards?

**Reply by dgafka** [reply]
@addisonschultz if you mean log in and out, then it does not help :)

**Reply by addisonschultz** [reply]
@dgafka Sorry! I meant if you re-authorized the app, by logging into **GitHub**. You can do this in the integration's configuration page

**Reply by dgafka** [reply]
Yep tried that, still the case :).
You actually see that by following documentation page: https://docs.ecotone.tech/quick-start-php-ddd-cqrs-event-sourcing/php-cqrs

**Reply by dgafka** [reply]
![Screenshot from 2023-07-10 19-48-57](https://github.com/GitbookIO/community/assets/6060791/1499791a-3c89-4e6e-a1f0-79482eaed33c)

**Reply by addisonschultz** [reply]
Sorry for the delay on this - but have been looking into this and think there might be something cached causing this to error out. 

Do you have many instances (blocks) of GitHub Files in your project? I wanted to see if you could 
- Remove the blocks
- Uninstall the integration
- Hard refresh
- Reinstall the integration
- See if new links are working

**Reply by dgafka** [reply]
Isn't this native feature? How can I uninstall integration with blocks?

**Reply by addisonschultz** [reply]
The GitHub Files block is a custom integration 😄 

You can find it here (in the organization it's not working in): https://app.gitbook.com/integrations/
